Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -853,6 +853,18 @@ namespace Babylon
                 {
                     m_position = Napi::Persistent(Napi::Object::New(info.Env()));
                     m_orientation = Napi::Persistent(Napi::Object::New(info.Env()));
+                    
+                    auto position = m_position.Value();
+                    position.Set("x", 0.f);
+                    position.Set("y", 0.f);
+                    position.Set("z", 0.f);
+                    position.Set("w", 1.f);
+
+                    auto orientation = m_orientation.Value();
+                    orientation.Set("x", 0.f);
+                    orientation.Set("y", 0.f);
+                    orientation.Set("z", 0.f);
+                    orientation.Set("w", 1.f);
```

Comment:
nit: slightly more efficient to do this

```suggestion
                    auto position{Napi::Object::New(info.Env())};
                    position.Set("x", 0.f);
                    position.Set("y", 0.f);
                    position.Set("z", 0.f);
                    position.Set("w", 1.f);
                    m_position = Napi::Persistent(position);

                    auto orientation{Napi::Object::New(info.Env())};
                    orientation.Set("x", 0.f);
                    orientation.Set("y", 0.f);
                    orientation.Set("z", 0.f);
                    orientation.Set("w", 1.f);
                    m_orientation = Napi::Persistent(orientation);
```